ios/RCTPSPDFKit/RCTPSPDFKitView.m

```diff
@@ -377,4 +434,14 @@ - (void)onStateChangedForPDFViewController:(PSPDFViewController *)pdfController
   }
 }
 
+- (NSArray <NSString *> *)buttonItemsStringFromUIBarButtonItems:(NSArray <UIBarButtonItem *> *)barButtonItems {
+  NSMutableArray *barButtonItemsString = [NSMutableArray new];
+  [barButtonItems enumerateObjectsUsingBlock:^(UIBarButtonItem * _Nonnull barButtonItem, NSUInteger idx, BOOL * _Nonnull stop) {
+    NSString *actionString = NSStringFromSelector(barButtonItem.action);
+    NSString *buttonNameString = [actionString stringByReplacingOccurrencesOfString:@"ButtonPressed:" withString:@"BarButtonItem"];
```

Comment:
This seems a bit fragile.

I wonder if it might make more sense if we alternatively check equality for all the supported bar button items of `PSPDFViewController`, and hardcode the strings for each. Could maybe be an RCTConvert as well.
If we also do this the other way around (from string to UIBarButtonItem), we could also get rid of the `valueForKey:` magic.

Although that would mean that we would need to extend this every time we add a new button item.

What do you think?

Reply:
Good point! Thank you! 👍 

I agree that the `valueForKey:` magic is seems fragile, but I think that it's preferable to the use hardcoded values. Because we will need to remember to update our wrapper every time we add or remove a `UIBarButtonItem` in the native SDK. I would also make renaming of a `UIBarButtonItem` more difficult.

So I would leave it as-is.

Reply:
On second thought, I'd prefer what you just suggested. That's because, even if we hardcode values in a converter, we'll also hardcode the `PSPDFViewController`'s `UIBarButtonItem` properties. And if we ever change or remove a `UIBarButtonItem` from `PSPDFViewController`, we'll get a compile error.

Thanks again, @steviki!

Reply:
Addressed in https://github.com/PSPDFKit/react-native/pull/233/commits/7d4b41b8215018a4af2127aacb4b7476b836f572